google/cloud/spanner/results.h

```diff
@@ -12,20 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_RESULT_SET_H_
-#define GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_RESULT_SET_H_
+#ifndef GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_RESULTS_H_
+#define GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_RESULTS_H_
 
-#include "google/cloud/spanner/internal/time.h"
 #include "google/cloud/spanner/row_parser.h"
 #include "google/cloud/spanner/timestamp.h"
 #include "google/cloud/optional.h"
 #include <google/spanner/v1/spanner.pb.h>
+#include <memory>
+#include <string>
+#include <unordered_map>
 
 namespace google {
 namespace cloud {
 namespace spanner {
 inline namespace SPANNER_CLIENT_NS {
-using QueryPlan = google::spanner::v1::QueryPlan;
+using ExecutionPlan = google::spanner::v1::QueryPlan;
```

Comment:
Since this is now a public symbol in our API, I think we should drop a little user-facing comment here about what an `ExecutionPlan` is. I don't think we need to duplicate all of QueryPlan's docs. but perhaps a brief sentence, and a link to QueryPlan's docs?

Reply:
Done.